kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -225,15 +225,24 @@ static function create( $contentObjectID )
         return new eZUser( $row );
     }
 
-    function store( $fieldFilters = null )
+    /**
+     * Only stores the entry if it has a Login value
+     *
+     * @param null $fieldFilters
```

Comment:
```suggestion
     * @param mixed|null $fieldFilters
```
Not much point in a param that is always null :)